src/app/shared/components/cluster-summary/template.html

```diff
@@ -41,7 +41,7 @@
        [fxFlex]="flipLayout ? '100%' : '50%'"
        fxLayout="column"
        class="step">
-    <div class="header counter-3">Settings</div>
+    <div class="header counter-2">Settings</div>
```

Comment:
Why is it changed? `Settings` step is 3rd and `Cluster` step is 2nd. It was ok.

Reply:
It was changed as the order in the dialog was 1 3 2 I believe.

Reply:
Instead of changing numbers it should be sorted, right?

Reply:
It should be based on `flipLayout` variable to make sure that in the wizard when there are 2 columns it will also be ok.

Reply:
cc @ahmadhamzh

Reply:
i checked if there 2 column and it looks good but there is some provider the summary for them don't show the settings and Initial Nodes so it's better to bee sorted am on it....

Reply:
I didn't check the code yet, but please note that the steps in the summary should have the same order as in the wizard. So first datacenter and provider etc.

Reply:
It's about this view
![image](https://user-images.githubusercontent.com/2285385/156196471-042c04f0-646b-4644-8211-e05b63098d3f.png)

If you change the counter then the step number will not match the section and it has to match. We could also simply disable the counter for the dialog mode and not display it at all.

Reply:
in the last commit below it will be like this :

![Screenshot from 2022-03-02 05-11-18](https://user-images.githubusercontent.com/85109141/156281512-ba234296-dd68-4580-8583-cfb2b09e4dc0.png)

and when create cluster from template  :

![Screenshot from 2022-03-02 05-14-34](https://user-images.githubusercontent.com/85109141/156281812-d64b5236-1569-4c72-addb-af1cf5d80a61.png)

and the summary for cluster if there is no settings and initial nodes will be like this :

![Screenshot from 2022-03-02 05-16-47](https://user-images.githubusercontent.com/85109141/156282120-b23262de-0ccb-4123-8e16-963bdab420b2.png)

so i think all are good now ??

Reply:
in the wizard summary i don't believe it could work like this
13
24
unless the dialog would look like it was
1
3
2
4
and also how can you disable the counter from just the dialog ??
the numbing come from the summary component which we call it in the dialog and in the wizard so i think its either we call the counter in both positions or disable it in both right ?
@floreks

Reply:
or the dialog could look like this 
![Screenshot from 2022-03-03 15-46-45](https://user-images.githubusercontent.com/85109141/156568814-6c08a5d5-9ec9-4aa9-9858-015c13b795eb.png)

Reply:
We could easily disable the counter inside the dialog and only show the title. There are a couple of different ways this could be done. One of them is passing a `mode` parameter to the component that defines if it works in standalone or dialog mode and then adjusting the UI based on that.